### PR TITLE
Fix win32 packaging

### PR DESCRIPTION
**Pull request by tolbrino**
Some system package updates require an automatic re-installation of styrene.

### CODE REVIEW

File: scripts/windows/msys2_prepare.bat [hunk 1]
```diff
@@ -15,6 +15,7 @@ IF "%OTP_VERSION%"=="" SET "OTP_VERSION=20.3"
 IF "%WIN_OTP_PATH%"=="" SET "WIN_OTP_PATH=C:\Program Files\erl"
 IF "%WIN_MSYS2_ROOT%"=="" SET "WIN_MSYS2_ROOT=C:\msys64"
 IF "%PLATFORM%"=="" SET "PLATFORM=x64"
+IF "%FORCE_SYRENE_REINSTALL%"=="" SET "FORCE_SYRENE_REINSTALL=false"
```
Comment (lucafavatella) [inline scripts/windows/msys2_prepare.bat]: SYRENE or STYRENE?

File: scripts/windows/msys2_prepare.bat [hunk 2]
```diff
@@ -89,9 +90,11 @@ START "" /WAIT "%TMP%\%OTP_PACKAGE%" /S
 @echo Current time: %time%
 rem Ensure Styrene is installed
 
-IF EXIST "%WIN_STYRENE_PATH%" GOTO STYRENEINSTALLED
+IF EXIST "%WIN_STYRENE_PATH%" IF "%FORCE_SYRENE_REINSTALL%" NEQ "true" GOTO STYRENEINSTALLED
```
Comment (lucafavatella) [inline scripts/windows/msys2_prepare.bat]: Though spelling is consistent.

File: scripts/windows/msys2_env_build.sh [hunk 3]
```diff
@@ -35,13 +35,13 @@ obe_otp_64_gcc_vsn_map="
     .*=>default
 "
 
-MSYS2_ROOT=${MSYS2_ROOT:-"${C_DRV}/msys64"}
```
Comment (lucafavatella) [inline scripts/windows/msys2_env_build.sh]: I see - C_DRV used here but defined later. Similar for other line.

File: scripts/windows/msys2_prepare.bat (hunk 1 above)
Comment (tolbrino) [reply scripts/windows/msys2_prepare.bat]: Fixed.

**Comment by tolbrino** [comment]
This PR is replaced by #2277

**Pull request closed without merging**